sherpa/plot/__init__.py

```diff
@@ -511,6 +511,58 @@ def point(self, x, y, overplot=True, clearwindow=False, **kwargs):
                      **opts)
 
 
+class Image(NoNewAttributesAfterInit):
+    """Base class for image plots
```

Comment:
[not really a comment that needs code changes]

One of the reasons I haven't really pushed the "plot image" work is because I feel we haven't come to a conclusion about how to handle 2D data. My feeling is that `Data2D` is for "unstructured" data whereas `DataIMG` is for a 2D matrix (aka an image), **but** this is not entirely born out by the code (e.g. `Data2D` has a `shape` setting), and then there's things that could get moved to `Data2D` - such as image filtering - that is orthogonal to the data representation.

So I am concerned that this design helps to cement this confusion. Fortunately

a) you've labelled it experimental
b) it is only used in one specific place

so it shouldn't break too much if we decide to rework the `Data2D`/`DataIMG` classes in the future.